sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
+maintenance of cellular integrity, metabolism, transcription/translation, and 
+cell-cell communication. Complete description of protein structures and 
+functions is a fundamental step towards understanding biological life and 
+also highly relevant in the development of therapeutics and drugs. Tons of 
+protein sequences have been generated, but fewer than 100,000 of them 
+have experimentally-solved structures. As a result, computational structure 
+prediction is essential for a majority number of protein sequences. However, 
+predicting protein 3D structures from sequence alone is very challenging, 
+especially when similar templates are not available. In the past decades, 
+various computational methods have been developed to predict protein 
+structure from different aspects, including prediction of secondary structure, 
+torsion angles, solvent accessibility, inter-residue contact map, disorder 
+regions and side-chain packing.
+
+Machine learning is extensively applied to predict protein structures and 
+some success has been achieved. For example, secondary structure can be 
+predicted with about 80% of Q3 accuracy by a 2-layer neural network 
+method PSIPRED [@doi:10.1093/bioinformatics/16.4.404]. Starting from 
+2012, deep learning has been gradually introduced to protein structure 
+prediction. The adopted deep learning models include deep belief network, 
+LSTM(long short-term memory), deep convolutional neural networks (DCNN) 
+and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1 
+@doi:10.1038/srep18962]. Here we focus on deep learning methods for 
+two representative subproblems: secondary structure prediction and 
```

Comment:
> Can you provide any intuition on why we want to focus on these subproblems? For example, is it where deep learning has had the biggest advantage over competitors or greatest success in an absolute sense?

> 1) we have something to say instead of just enumerating methods 

Completely agree!

I think adding something similar to what @agitter outlined ("Starting with 2012..." also similar to @j3xugit's [comment above](https://github.com/greenelab/deep-review/pull/191/files#r98305715)) and being concise about how deep learning has indeed remarkably improved performance over time (and citing papers with their architectures) may be preferable to enumerating methods in as great detail. 

I think the next two or three paragraphs could probably be combined if we adopt this strategy. As is, I think this section may be a bit long if we want to squeeze in other subsections within `Study`.

> 2) deep learning is contributing something unique, so this sounds great to me.

I agree that deep learning is improving performance, but, to play devil's advocate, who cares?

Perhaps it would be more pertinent and in line with what has been written in the section about [morphological phenotypes](https://github.com/greenelab/deep-review/blob/master/sections/04_study.md#morphological-phenotypes) and in the [rough outline of the study section](https://github.com/greenelab/deep-review/blob/master/sections/04_study.md#how-is-deep-learning-used-to-study-basic-biological-processes-in-a-manner-that-may-provide-future-insights-into-human-disease) if we follow with a discussion about three forward-looking points: 

* what knowledge of protein structure leads to
* how biomedical applications may be impacted by this knowledge
* if deep learning approaches can get us to that point in the future

This could be added as a final short paragraph of the subsection if this is what we feel the main message for the review is.